src/main/java/com/icthh/xm/ms/entity/web/rest/LinkResource.java

```diff
@@ -39,6 +41,7 @@
  * REST controller for managing Link.
  */
 @Slf4j
+@Transactional
```

Comment:
O my god. Iconic antipattern.

Why it is not in service?

Reply:
![image](https://github.com/user-attachments/assets/6054e7f8-1b65-494e-bb52-46f5bf46e922)
it is in a servce, but it doesn't help